website_sale_brand_catalogs/views/pages/catalogs.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <!-- Brands catalog -->
+    <template id="brand_catalog" name="Catalogs">
+        <t t-call="website.layout">
+            <div id="wrap">
+                <t t-set="additional_title" />
+                <div class="oe_structure oe_empty" id="oe_structure_website_catalogs_1" />
+                <div class="oe_structure oe_empty" id="oe_structure_website_catalogs_2">
+                    <div id="brand_catalog" class="container mt24 mb24 catalog_brand_page bg-white">
+                        <div class="col-sm-12">
+                            <h2 class="text-center mt32 mb24 text-uppercase font-weight-bold">Catalogs by brand</h2>
+                        </div>
+                        <div class="row">
+                            <div t-if="not catalog_brands" class="col-sm-12">
+                                <h2 class="text-center mt48 mb32">No catalogs available</h2>
+                            </div>
+                            <t t-else="" t-foreach="catalog_brands" t-as="brand">
```

Comment:
I think the `t-else` is not required. If the variable is empty, no iterations will be performed.

I'd move the above t-if below the t-foreach

Reply:
I use it to display the message that there are no catalogs.

Reply:
I mean:

```
<t t-foreach="catalogs" t-as="catalog">
    ...
</t>
<t t-if="not catalogs">
    ...
</t>
```